style.css

```diff
@@ -0,0 +1,784 @@
+    margin-bottom: 30px;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .past-events div {
+    display: flex;
+    column-gap: 20px;
+  }
+
+  .activity img {
+    display: flex;
+    margin: 0 auto;
+    width: 50px;
+    height: 50px;
+  }
+
+  .footerUpper hr {
+    width: 35px;
+    color: #ec5242;
+  }
+
+  .footerLower img,
+  .footerLower1 img {
+    width: 150px;
+    height: 150px;
+  }
+
+  .footerLower1 img {
+    color: #fff;
+    margin-right: 20px;
+  }
+
+  .speakerList img {
+    width: 200px;
+    border-radius: 100%;
+  }
+
+  .past-events div img {
+    width: 400px;
+    height: 250px;
+    opacity: 0.2;
+  }
+
+  /* End of about page */
+  .introContainer {
+    margin: 0 15% 5% 15%;
+  }
+
+  .activity p {
+    text-align: center;
+  }
+
+  .footerLower p {
+    margin: 7px;
+  }
+
+  .introContainer p:nth-child(1) {
+    font-size: 50px;
+    color: #ec5242;
+  }
+
+  .introContainer h1 {
+    font-size: 75px;
+    color: #ec5242;
+    font-weight: 900;
+    padding-bottom: 15px;
+  }
+
+  .introContainer h2 {
+    padding: 0 0 10px 0;
+    font-size: 45px;
+    color: #272a31;
+    font-weight: 900;
+  }
+
+  /* Main Section */
+  .main {
+    background-image: url("./Images/diamond.png");
+    background-repeat: repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #272a31;
+    color: white;
+  }
+
+  .main h1 {
+    color: white;
+    font-size: 40px;
+    font-weight: 500;
+  }
+
+  .main hr {
+    width: 400px;
+    color: #ec5242;
+    margin: 5px 0 60px 0;
+  }
+
+  .mainActivities {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    gap: 10px;
+    margin: 50px 70px 80px 70px;
+  }
+
+  .activity {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #272a35;
+    padding: 30px 10px;
+  }
+
+  .activity h2 {
+    color: #ec5242;
+    padding: 20px 0;
+  }
+
+  .seeHeader {
+    display: flex;
+    margin-top: 50px;
+    font-size: 25px;
+  }
+
+  .underline {
+    display: flex;
+    width: 1100px;
+    color: #272a31;
+    margin-bottom: 30px;
+  }
+
+  .speakers {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .speakers hr {
+    margin-top: 5px;
+  }
+
+  .speaker-content {
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-top: 30px;
+  }
+
+  .speaker-content div {
+    margin-top: 30px;
+    column-gap: 20px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .speakerList {
+    display: flex;
+  }
+
+  .speakerList div {
+    width: 100%;
+    margin: 0 30px 7px 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .speakerList div hr {
+    width: 20px;
+    margin-bottom: 10px;
+  }
+
+  .para {
+    color: #ec5242;
+    padding: 10px 0;
+  }
+
+  /* Footer */
```

Comment:
- [ ] Kindly add some extra gaps/spaces between the `past-events` section and the `footer` to match the given design.
  - Your design:
  - ![Screenshot from 2023-02-09 22-16-02](https://user-images.githubusercontent.com/60083437/217871116-6fcbe4ec-52dc-4305-a17f-08f64000cfd6.png)

  - Given design:
  - ![Screenshot from 2023-02-09 22-13-15](https://user-images.githubusercontent.com/60083437/217870973-67fe0768-509f-4f8d-9ee7-d907c745b9ee.png)

Reply:
Okay 
I will fix the gap issues in the project.